sources/hachiko/src/main.rs

```diff
@@ -0,0 +1,66 @@
+mod config;
+mod error;
+use crate::config::Config;
+use crate::error::Result;
+use imdsclient::ImdsClient;
+use snafu::{OptionExt, ResultExt};
+use std::fs;
+use log::{error, trace, info, warn};
+use log::LevelFilter;
+use simplelog::{Config as LogConfig, SimpleLogger};
+use tokio::time::{sleep, Duration};
+
+/// Path to config file containing host's autoscaling.should-wait setting
```

Comment:
I'm actually not sure if docstrings work for constants. You can check by running `cargo doc` and peeking at the output. I tend to just use regular comments.

```suggestion
// Path to config file containing host's autoscaling.should-wait setting
```